### ISSUE TITLE
Done in: __days on:__ . Not working.

### ISSUE THREAD

**Issue by ghost** [Kazuwuqt/Anki_More_Overview_Stats#8]
Hey, thank you for this great addon.

The Done in: (The last row statistic) is showing wrong value. Right now i am using Anki latest version 24.0+ and on a Mac. and Show table for finished dekcs: true, isn't working. It's just showing Congratulations screen. 
![image](https://github.com/user-attachments/assets/5cfeb90d-7e60-4ac9-9337-68ed8b87054c)

![image](https://github.com/user-attachments/assets/c37de1df-da05-454e-b97f-2a027dced814)

**Comment by patrick-mahnkopf** [comment]
Hi and thank you for opening this issue.

\
Anki seems to have recently made some changes that prevented the stats table from showing up on finished decks.
I have just fixed this in the newest add-on version, so please try updating and let me know if the issue persists.

\
I also checked the `Done in` stats you mentioned, but everything seems to work correctly.
Please note that the last row uses the `New cards/day` value from the `Daily Limits` in your deck's options to calculate the remaining days left to finish the current deck.

This means that if you have a limit of 50 or more set for the deck shown in your first screenshot, you should be done after studying all cards up until that limit for another day.
In other words, if the limit for the deck shown in your screenshot is at or above 50, all values in your screenshot for this deck are correct.

Example:
Your deck has `50` new cards left.
Your `New cards/day` setting is at or above `50`.
Studying until that limit of `50` for `1` more day will cause you to finish the deck.
Thus, the stats table would show: `Done in: 1 day` and tomorrow's date.

Example 2:
Your deck has `50` new cards left.
Your `New cards/day` setting is set to `10`.
Studying until that limit of `10` every day will cause you to finish the deck in `5` days.
Thus, the stats table would show: `Done in: 5 days` and the corresponding date.

**Comment by ghost** [comment]
Hey, thanks for the update. Really love the addon. 
For the Done in date, there is a peculiar issue. 
We can set new cards for a preset in 2 ways(for the entire preset or per a deck). When i placed it for the entire preset, the done in date is working exactly as it should. But if i place per deck new cards limit in the preset(let's say 5) then it is considering the number of new cards in that deck as 5 only(inspite there being 800+). 
So daily review limit = 5. Number of new cards in that deck = 5. 5/5 = 1 day. 
![Anki 2024-08-21 09 20 26](https://github.com/user-attachments/assets/93885809-7d29-47f7-97e1-da92700a5a95)
![Anki 2024-08-21 09 20 32](https://github.com/user-attachments/assets/af13ef7a-de7f-4f2d-92e4-f42eeae412fc)


Here in the above image you can see number of unseen cards as 800+, but number of new cards is only 5. Would this work? instead of new cards/ daily limit, using unseen cards/daily limit might actually give the result in any case.

**Comment by patrick-mahnkopf** [comment]
Hi again, and thank you for the kind words!

I have just added support for the deck-specific daily limit settings.
This functionality wasn't available yet when I last updated the add-on, which is why it didn't work correctly yet.

Please update to the latest version and let me know if this resolves your issue.
\
\
For clarity:
- The 'New' row displays cards up to the limit set for this specific deck.
- The 'Unseen' row represents the total number of new cards in this deck, regardless of deck limits.


The add-on was already using the `unseen cards / daily limit` calculation, but didn't account for deck-specific limits.
This update should also make the 'Done in' stats show correct values now.

**Comment by ghost** [comment]
Thank you. This is perfect.
